source/class/qx/tool/cli/commands/Test.js

```diff
@@ -156,29 +151,35 @@ qx.Class.define("qx.tool.cli.commands.Test", {
       this.argv.watch = false;
       this.argv["machine-readable"] = false;
       this.argv["feedback"] = false;
+      this.argv["show-startpage"] = false;
       // check for special test compiler config
       if (!this.argv.configFile && fs.existsSync(path.join(process.cwd(), qx.tool.cli.commands.Test.CONFIG_FILENAME))) {
         this.argv.configFile = qx.tool.cli.commands.Test.CONFIG_FILENAME;
       }
       this.addListener("making", () => {
-        if (!this.hasListener("runTests")) {
+        if (!this.hasListener("runTests") && (this.__tests.length === 0)) {
           qx.tool.compiler.Console.error(
-            `No test runner registered!
+            `No tests are registered!
                Please register a testrunner, e.g. testtapper with:
                qx package install @qooxdoo/qxl.testtapper
+               See documentation https://qooxdoo.org/docs/#/development/testing/
```

Comment:
See documentation **at** https:/

Reply:
Oh, errorCode is gone. Testtapper has to be fixed after release